cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-january-2023.adoc

```diff
@@ -0,0 +1,682 @@
+
+
+//GH#37154
+|*Added Support for Tanzu Application Service (TAS) on Windows*
+|You can now deploy Defenders to scan your Windows TAS environments.
+The Defenders are deployed as addon software on the Windows Diego cells of your TAS environment, which is similar to how they are deployed on Linux. You must now select the Orchestrator deployment method to deploy the TAS Defenders. Defenders on Windows TAS environments don't support the following features.
+
+* Scan of applications running Docker images on TAS
+* Use of a proxy to install a tile
+* Cert-based authentication
+* Blobstore scanning: Defenders on Windows can't be scanners and Windows droplets have no results.
+
+//GH#37772
+|*New Fields to Splunk Alerts*
+|The following https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin-compute/alerts/splunk[fields are added] to Splunk alerts.
+
+* `command` - Shows the command which triggered the runtime alert.
+* `namespaces` - Lists the Kubernetes namespaces associated with the running image.
+* `startup process` - Shows the executed process activated when the container is initiated.
+
+//[GH#36775] 
+|*In-Depth Scanning of Nested Java Archives*
+|In previous releases, Defenders scanned two levels deep in nested https://docs.oracle.com/javase/8/docs/technotes/guides/jar/jarGuide.html[Java Archives] (JARs).
+The latest version of Defender can scan up to ten levels of nested JARs.
+While this level of nesting is atypical, this capability improved the scan accuracy by detecting the vulnerabilities in the deepest nested jars.
+You can view the vulnerabilities in your images with the following steps.
+
+. Go to *Monitor > Vulnerabilities > Images*.
+. Filter the results to show your packages using JARs.
+. Click on the shown results to see the details.
+. Go to Package info and filter the results.
+
+//GH#32746 |
+|*Twistcli Sandbox for Third-Party Assessment Tools*
+|To help you augment and expand the compliance checks the  twistcli sandbox now enables you  to run a third-party binary/script of choice within the sandboxed container.
+
+For example: `./twistcli sandbox --token "token" --volume /opt/sandbox_testing_tools:/opt/sandbox --analysis-duration 0.1m --third-party-delay 0.2m --third-party-cmd "/opt/sandbox/test_tool" --third-party-output /opt/sandbox/output.txt --v <image:tag>` 
+
+You can view the scan results on the mounted volume and on "Monitor > Runtime > Image analysis sandbox". 
+In this example the output of the 3rd party testing tool will be written to the `/opt/sandbox_testing_tools/output.txt file` on the sandbox host.
+
+// CWP-41281 (Update1)
+|*Twistcli for ARM64 Mac*
+|twistcli is now supported on ARM64 Mac machines.
+
+Download the ARM64 Mac-compatible version of twistcli from "Manage > System > Utilities", or using the API `/util/osx/arm64/twistcli`.
+
```

Comment:
@divyabhushan we are missing 2 features here, that are a part of Lagrange Update 1 (they are mentioned in the PCCE release notes):
1. Filter Defenders by TAS Foundation ID https://redlock.atlassian.net/browse/CWP-42365
2. Support for Talos Linux - container vulnerabilities and compliance https://redlock.atlassian.net/browse/CWP-42198

Reply:
@grevach Added these two features. Thanks.

Reply:
@grevach I've added a mention on sorting by foundation for TAS on the Windows support RN and think that we should include it on the TAS defender deployment flow. WDYT?
I've also added the limitations regarding Talos Linux. Once support is more robust, we might need to create a deployment flow for it too, but IMO we don't need to do it now.
Thanks @divyabhushan and let me know if there are any other RN updates you need from me.